src/components/header/header.tsx

```diff
@@ -0,0 +1,49 @@
+import { useLocation, useNavigate } from "react-router-dom";
+import "./header.css";
+import ImgHomeLogo from "../../assets/logo/img_logo.svg";
+
+function Header() {
+  const location = useLocation();
+  const navigate = useNavigate();
+  const isCommunityPage = location.pathname === "/board";
+  const isMarketPage = location.pathname === "/items";
```

Comment:
```suggestion
  const location = useLocation();
  const navigate = useNavigate();
  
  const isCommunityPage = location.pathname === "/board";
  const isMarketPage = location.pathname === "/items";
```
NIT: 매우 마이너한 코멘트라서 반영 안하셔도돼요! 
저라면 hooks를 사용한 코드와 변수를 만드는 로직간에 연결성이 없으므로 위와 같이 한칸을 띄워줄것같아요.
(서로 관련 있는 코드들을 묶고, 관련성이 적은 코드들 사이에 빈 줄을 두어 가독성을 높일수 있게끔!)

이런식으로 리뷰어를 위해 코드를 깔끔히 정리해나가며 작성하는 습관을 들이는 연습도 해보시면 좋답니다 :)
리뷰어가 다른 개발자가 될수도있고, 혹은 추후에 유지보수를 고려할때 코드를 수정하는 당사자가 채연님이 될수도있으니까요!